projects/components/src/quick-search/quick-search.component.html

```diff
@@ -42,7 +42,10 @@ <h5 *ngIf="showSectionTitle(searchSection)" class="search-result-section-title">
                     >
                         {{ item.displayText }}
                     </li>
-                    <li class="no-results" *ngIf="searchSection.result?.items?.length === 0">
+                    <li
+                        class="no-results"
+                        *ngIf="searchSection.result?.items?.length === 0 && showSectionTitle(searchSection)"
```

Comment:
This is a pretty complex statement, consider putting it into a method, there's already a method call